datalad/interface/run_procedure.py

```diff
@@ -181,15 +184,16 @@ def _guess_exec(script_file):
     # TODO check for exec permission and rely on interpreter
     if is_exec:
         return {'type': u'executable',
-                'template': u'"{script}" "{ds}" {args}',
+                'template': u'{script} {ds} {args}',
                 'state': state}
     elif script_file.endswith('.sh'):
         return {'type': u'bash_script',
-                'template': u'bash "{script}" "{ds}" {args}',
+                'template': u'bash {script} {ds} {args}',
                 'state': state}
     elif script_file.endswith('.py'):
+        ex = sys.executable if on_windows else shlex_quote(sys.executable)
```

Comment:
Can also be `maybe_shlex_quote(sys.executable)` now

Reply:
good point, thx.